commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/models/TypeUtils.java

```diff
@@ -89,6 +89,23 @@ public ClassName getBuilderType(final TypeElement typeElement) {
                 .filter(this::isPropertyMethod);
     }
 
+
+    /**
+     * Returns all property methods as stream.
```

Comment:
Here we should also update the copied javadoc - because this method only returns the direct properties of the given `TypeElement`. Additionally you should make the parameter `final` and check the original method too - looks like I forgot to add the `final` 😉